README.md

```diff
@@ -609,10 +609,10 @@ This will add `my-custom-property` and `my-custom-property2` to the metadata of
                ...
          transform-options:
            table-properties:
-            my-custom-param: my-custom-value
-            my-custom-param2: my-custom-value2
+            my-custom-property: my-custom-value
```

Comment:
if I specify 1 properties in my table replication and I also specify 1 different property in the "global" one. I think from looking at the code the 1 table replication property is then ignored. Is this what we want?

Reply:
From the readme, any properties in a table replication will override the global properties. Where in the code do you see that it is ignored? 

On [line 44 in AbstractTableParametersTransformation](https://github.com/HotelsDotCom/circus-train/pull/156/files#diff-b75474f19d562493acfb0d036ff0178cR44) the override is returned if they exist, instead of the global.

Reply:
that's my point it's either all global ones or all local ones. I can't set 2 global properties and override just 1 in a replication. Wouldn't it make more sense to take the whole local map and add all override key/values??

Reply:
Hmm.. thinking from a user perspective, how will he set certain properties for his one particular table when he doesn't want to add global properties ?

I incline towards letting user copy some of the global properties to table level if he needs both, at least then we support both use-cases.

Reply:
ok I agree, in the alternative you'd have to override and set to null or something. All good.